addons/mail/static/src/components/emoji_picker_view/emoji_picker_view.scss

```diff
@@ -0,0 +1,4 @@
+.o_EmojiPicker {
+    height: Min(80vh, 350px);
```

Comment:
With current layout, 340px or a bit less would be better to cut the last line (when > 9 lines) in the middle rather than just below.

To be kept up to date depending on future spacing changes in the grid.

Maybe the height should be on the grid directly, and let the rest of the popover take its natural space for the category?

Also when doing this, I notice the emojis overflow above the border of the popover

Reply:
Maybe the value could be computed from other scss variables such as spacers and font height to actually match what we need?